### PR TITLE
docs(configuration): New docs for creating AAD policty and fhirUser

### PR DESCRIPTION
**Pull request by erikhoward**
Add docs for creating customer AAD ExtensionAttribute and fhirUser policies.

### CODE REVIEW

File: samples/Patient and Population Services G10/docs/deployment.md [hunk 1]
```diff
@@ -186,6 +186,59 @@ The EHR Launch application is a standard confidential client application which l
 ![](./images/deployment/5_ehr_confidental_app_scopes.png)
 </details>
 
+#### SMART FHIRUser Custom Claim
+
+#### Prerequisites
+
+You must have rights to administer claims policy in your Azure Active Directory Tenant and read/write permissions for user profiles in order to proceed.
```
Comment (yoju0121) [inline samples/Patient and Population Services G10/docs/deployment.md:193]: Nit: I think detailing how one can check this would be helpful

**Comment by mikaelweave** [comment]
We discussed today, but the extensionAttributes should be on the users vs the application. Also to get the fhirUser into the access token, this change will need to be on the Azure Healthcare APIs enterprise application. We should advise our users how to come up with an organizational policy around this. Not sure how this impacts ID token generation tbh.

File: samples/Patient and Population Services G10/docs/deployment.md [hunk 2]
```diff
@@ -186,6 +186,59 @@ The EHR Launch application is a standard confidential client application which l
...
+
+Connect-MgGraph -TenantId Your-Tenant-Id -ContextScope Process -Scopes $scopes
+
+$params = @{
+  OnPremisesExtensionAttributes = @{
+    Your-Extension-Attribute-Name = "selected-patient-fhir-id"
+  }
+}
+
+Update-MgUser -UserId Your-Enterprise-Application-ObjectId -BodyParameter $params
```
Comment (namalu) [inline samples/Patient and Population Services G10/docs/deployment.md:239]:
```suggestion
Update-MgUser -UserId Your-User-ID -BodyParameter $params
```

Is this supposed to be user id?

**Comment by mikaelweave** [comment]
@erikhoward can we replace some of the scripts with a Portal UI Experience?

- Azure AD Powershell library isn't needed
- Only the last script (that uses `Microsoft.Graph.Users`) is needed.
- Docs on the Claims Mapping in the portal: https://learn.microsoft.com/en-us/azure/active-directory/develop/active-directory-jwt-claims-customization

**Pull request closed without merging**